### PR TITLE
Add docx output

### PR DESCRIPTION
**Pull request by jashapiro**
### Purpose

Modified the github action to build the docx file as well as html and pdf outputs.

### Issue

_What GitHub issue does your pull request address?_

#298

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

In theory this should allow for a `.docx` output to appear in the `output` branch, and I would expect to see the same in the `artifacts` of the build.

#### Which areas should receive a particularly close look?

How does the output look?

#### Is the pull request ready for review?

Let's see what happens with the first run.

### CODE REVIEW

**Comment by jashapiro** [comment]
Looks like this works, mostly, though the frontmatter is kind of a mess (Word support for SVGs seems... lacking)

**Comment by jashapiro** [comment]
Marking this ready for review. I replaced the icon svgs with pngs, which solved the awful docx rendering, at the cost of slightly less pretty pdf & html output. 

Otherwise, the docx rendering looks fine, and I added it to the appveyor script for one-click download of the docx version (e.g. https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/302#issuecomment-1211244567).

I believe that the github action update means the output branch will get the docx file on merge as well, but somebody might want to confirm that.

File: content/06.methods.md [hunk 1]
```diff
@@ -292,7 +292,7 @@ Based on pediatric cancer literature review, we added _MYBL1_ [@doi:10.1073/pnas
...
+We collated canonical genes from the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors: ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors [@https://doi.org/10.1007/s00381-017-3481-3; @doi:10.1016/j.ccell.2015.04.002; @doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @doi:10.1016/s0002-9440(10)64477-x; @doi:10.1016/j.jaad.2017.05.059; @doi:10.1186/s40478-020-01056-8].
```
Comment (jashapiro) [inline content/06.methods.md:295]: This is the only non-whitespace change in this PR, but the current version still does not get the reference properly for some reason. Will try one more change.

Comment (jharenza) [reply content/06.methods.md:295]: I have found that PMID works in these cases of long strings of dois which do not sometimes render

Comment (jashapiro) [reply content/06.methods.md]: Turns out it was a different reference that was causing the trouble: Manubot can't handle parenthesis in a doi.

File: content/06.methods.md [hunk 2]
```diff
@@ -292,7 +292,7 @@ Based on pediatric cancer literature review, we added _MYBL1_ [@doi:10.1073/pnas
...
+We collated canonical genes from the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors: ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors [@pmid:28623522; @doi:10.1016/j.ccell.2015.04.002; @doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @pmid:12466115; @doi:10.1016/j.jaad.2017.05.059; @doi:10.1186/s40478-020-01056-8].
```
Comment (jashapiro) [inline content/06.methods.md:295]: Non-whitespace change

File: content/06.methods.md [hunk 3]
```diff
@@ -448,66 +448,66 @@ If this filtering still resulted in multiple specimens, we randomly selected a s
 
 #### Quantification of Telomerase Activity using Gene Expression Data (`telomerase-activity-prediction` analysis module)
 
-We predicted telomerase activity of tumor samples using the recently developed `EXTEND` method [@doi:https://doi.org/10.1101/2020.05.21.109249].
-Briefly, `EXTEND` estimates telomerase activity based on the expression of a 13-gene signature. 
-We derived this signature by comparing telomerase-positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity. 
+We predicted telomerase activity of tumor samples using the recently developed `EXTEND` method [@doi:10.1038/s41467-020-20474-9].
```
Comment (jashapiro) [inline content/06.methods.md:451]: Updated reference (now published!)

File: content/06.methods.md [hunk 4]
```diff
@@ -448,66 +448,66 @@ If this filtering still resulted in multiple specimens, we randomly selected a s
...
+Next, we used multivariate Cox (proportional hazards) regression analysis [@doi:10.1111/j.2517-6161.1972.tb00899.x] to model the following: a) `tp53 scores + telomerase scores + extent of tumor resection + LGG group + HGG group`, in which `tp53 scores` and `telomerase scores` are numeric, `extent of tumor resection` is categorical, and `LGG group` and `HGG group` are binary variables indicating whether the sample is in either broad histology grouping, b) `tp53 scores + telomerase scores + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), and c) `quantiseq cell type fractions + CD274 expression + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), in which `quantiseq cell type fractions` and `CD274 expression` are numeric.
```
Comment (jashapiro) [inline content/06.methods.md:459]: Found DOI for this ref.

File: content/06.methods.md (hunk 1 above)
Comment (jharenza) [reply content/06.methods.md]: Oh, Manubot.

**Review by jashapiro — COMMENTED** [review]
Noting below which lines in the methods are not just whitespace changes

**Review by jharenza — APPROVED** [review]
This looks good, and no more `???`!

**Comment by jharenza** [comment]
The one thing I noticed and wanted to note somewhere is that the reference format is general Manubot format and I am wondering if there will be a way to make the references in the output journal-specific. I can add a new ticket around this, as I think we do not need to address immediately nor prior to sending to co-authors.

**Comment by jharenza** [comment]
Update: created #304 for reference style.

**Review by jaclyn-taroni — APPROVED** [review]
:rocket:

**Pull request merged**